offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -797,6 +1094,39 @@ export async function parseBurnTx(tx: Transaction): Promise<RecipientCellData |
   return cellData;
 }
 
+export function checkLock(amount: bigint, assetIdent: string, txHash: string, ckbLock: CkbLock): string {
+  const nervosAsset = ForceBridgeCore.config.eth.nervosAssetWhiteList.find(
+    (asset) => asset.typescriptHash === assetIdent,
+  );
+  if (!nervosAsset) {
+    return `nervos asset ${nervosAsset} not in nervos asset white list`;
+  }
+  const minimalAmount = nervosAsset.minimalBridgeAmount;
+  const bridgeFeeFromConfig = BigInt(ForceBridgeCore.config.eth.lockNervosAssetFee);
+  const bridgeFeeSaved = BigInt(ckbLock.bridgeFee);
+  if (assetIdent == '0x0000000000000000000000000000000000000000000000000000000000000000') {
+    // lock ckb
+    if (
+      BigInt(ckbLock.amount) <= bridgeFeeFromConfig ||
+      BigInt(ckbLock.amount) - bridgeFeeFromConfig < BigInt(minimalAmount)
```

Comment:
minimalAmount is always configed greater than bridgeFee, so  we only check ckbLock.amount >= minimalAmount here

Reply:
fixed